<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDispose</class>
 <widget class="QMainWindow" name="FileDispose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileDispose</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditPath"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonPath">
        <property name="text">
         <string>Preview..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Del Context:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditDelContext"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonDel">
        <property name="text">
         <string>Delelte</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionExit_Q"/>
   </widget>
   <widget class="QMenu" name="menuTool_T">
    <property name="title">
     <string>Tool(&amp;T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout_A">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout_A"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool_T"/>
   <addaction name="menuAbout_A"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit_Q">
   <property name="text">
    <string>Exit(&amp;Q)</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="filedispose.qrc"/>
 </resources>
 <connections/>
</ui>
